### PR TITLE
Tooltip improvements

### PR DESCRIPTION
**Pull request by daiagi**
**Thank you for your contribution** to the [KodaDot - One Stop Shop for Polkadot NFTs](https://kodadot.xyz).

👇 __ Let's make a quick check before the contribution.

## PR Type

- [x] Bugfix
- [ ] Feature
- [x]  Refactoring

## Context

- [x] fix isssue where tooltip was not above content (works only if append-to-body:false)
- [x] font-size prop on NeoTooltip (works only if append-to-body:false)
- [x] add multiline option
- [x] refactor scss
- [x] make append-to-body prop false by default

#### Before submitting pull request, please make sure:

- [x] My contribution builds **clean without any errors or warnings**
- [x] I've merged recent default branch -- **main** and I've no conflicts
- [x] I've tried to respect high code quality standards
- [x] I've didn't break any original functionality

#### Optional

- [ ] I've tested it at </ksm/collection>
- [ ] I've tested PR on mobile
- [ ] I've written unit tests 🧪
- [ ] I've found edge cases

#### Had issue bounty label?

- [x] Fill up your KSM address: [Payout](https://beta.kodadot.xyz/transfer/?target=EfmnRhHaQqfT3phm4cUCHCU3gFVDoSPR1U9WXzMRQBMqZ4L)

#### Community participation

- [ ] [Are you at KodaDot Discord?](https://discord.gg/35hzy2dXXh)

## Screenshot 📸

- [ ] My fix has changed **something** on UI; a screenshot is best to understand changes for others.

### <samp>🤖 Generated by Copilot at 932aa85</samp>

Refactored and enhanced the `NeoTooltip` component to use `Oruga` and added new props and style options. Updated the usage of the `append-to-body` prop in various components to improve tooltip visibility and avoid overlaps.

### <samp>🤖 Generated by Copilot at 932aa85</samp>

> _Sing, O Muse, of the skillful coder who changed the NeoTooltip_
> _And made it more flexible and elegant with Oruga's help_
> _He removed the append-to-body prop from many a component_
> _And added it where it was needed, with wisdom and intent_

### CODE REVIEW

**Review by preschian — APPROVED** [review]

File: components/gallery/GalleryItemAction/GalleryItemActionType/GalleryItemTransfer.vue [hunk 1]
```diff
@@ -6,6 +6,7 @@
         <template #action>
           <NeoTooltip
             :active="isTransferButtonDisabled"
+            :append-to-body="true"
```
Comment (vikiival) [inline components/gallery/GalleryItemAction/GalleryItemActionType/GalleryItemTransfer.vue:9]:
```suggestion
            append-to-body
```

Comment (vikiival) [reply components/gallery/GalleryItemAction/GalleryItemActionType/GalleryItemTransfer.vue:9]: ?

Comment (daiagi) [reply components/gallery/GalleryItemAction/GalleryItemActionType/GalleryItemTransfer.vue:9]: > ?

Oh yeah..🤦

**Review by vikiival — APPROVED** [review]

**Comment by vikiival** [comment]
Hello, can you please resolve conflicts?

**Pull request merged**